src/dashboards/Map/BicycleTag/styles.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../../assets/styles/import.scss';
+@import '../../../variables.scss';
+
+.bicycle-tag {
+    background-color: #388f76;
```

Comment:
Bruk helst `$colors-transport-default-mobility` for denne fargeverdien. https://design.entur.org/komponenter/ressurser/tokens#farger

Reply:
👍